container/docker.go

```diff
@@ -45,6 +46,20 @@ func (d dockerRuntime) PID(id string) (uint32, error) {
 	return uint32(ci.State.Pid), nil
 }
 
+// MaybeExists returns false if we are sure we cannot find the container, true otherwise
+func (d dockerRuntime) MaybeExists(id string) (bool, error) {
```

Comment:
Why not just `Exists`? It's always a maybe if it can not exists 😅 .

You wanted to emphasis if it exists, it's still not necessarily true? Or the opposite? 🤔